atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SpecificTableSweeper.java

```diff
@@ -279,9 +292,17 @@ private void saveFinalSweepResults(TableToSweep tableToSweep, SweepResults final
         });
     }
 
-    private void reportSweepMetrics(SweepResults sweepResults) {
-        sweepMetrics.examinedCells(sweepResults.getCellTsPairsExamined());
-        sweepMetrics.deletedCells(sweepResults.getStaleValuesDeleted());
+    private void reportSweepMetricsAfterOneIteration(SweepResults sweepResults) {
+        sweepMetrics.examinedCellsOneIteration(sweepResults.getCellTsPairsExamined());
+        sweepMetrics.deletedCellsOneIteration(sweepResults.getStaleValuesDeleted());
+        sweepMetrics.sweepTimeOneIteration(sweepResults.getTimeInMillis());
```

Comment:
We should record time since start metric on each iteration too.  I know this goes up by 1 second per second so doesn't look that useful - but it allows us to create a monitor for "we've been sweeping the same table for 3 days and still haven't completed it".  If we just have this at the end of the sweep we may never see the metric if it is growing faster than we can sweep it.